src/vanna/qdrant/qdrant.py

```diff
@@ -305,7 +305,7 @@ def _setup_collections(self):
             )
         if not self._client.collection_exists(DOCUMENTATION_COLLECTION_NAME):
             self._client.create_collection(
-                name=DOCUMENTATION_COLLECTION_NAME,
+                collection_name=DOCUMENTATION_COLLECTION_NAME,
```

Comment:
I had passed this as a positional arg instead. Unsure where the `name` key came from. Thanks.